components/seo.jsx

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import Head from "next/head";
+
+const SEO = () => {
```

Comment:
I know you talked about this a little bit in the overview but, this is just to pull out the head tags? How does that affect SEO? (I know very little about SEO 😬 )

Reply:
Yep, just for the head tags! Doesn't really change anything from our current head tag setup, but just eliminates some duplicated titles/descriptions and allows us to take advantage of [Webfont optimization](https://nextjs.org/blog/next-10-2#automatic-webfont-optimization) 👀